_includes/package_body.html

```diff
@@ -83,6 +83,49 @@ <h4>Third-Party Package</h4>
                     {{package.data.pkg_type | upcase}}</span>
                 </td>
               </tr>
+              <tr>
+                <td class="text-right"><b>CI status</b></td>
+                <td style="position: absolute;">
+                  {% assign ci_data = package.data.ci_data %}
+                  {% if ci_data.ci_available %}
+                    {% if ci_data.history_available %}
+                      <div class="dropdown">
+                        <button type="button" data-toggle="dropdown" class="btn label label-primary dropdown-toggle" style="font-size: 75%; padding: .3em .6em .4em;border: 0;" title="{{ ci_data.tooltip }}">
+                          <span class="glyphicon glyphicon-check"></span>
+                          Continuous Integration
+                          {% if ci_data.stats_available %}
+                            : {{ ci_data.tests_ok }} / {{ ci_data.tests_total }}
+                          {% endif %}
+                          <span class="caret" style="margin: 0 3px 0 5px;"></span>
+                        </button>
+                        <ul class="dropdown-menu" role="menu">
+                          <li class="dropdown-header">Build history (last {{ ci_data.history.size }} of {{ ci_data.total_builds }} builds):</li>
+                          {% for build in ci_data.history %}
+                            <li>
+                              <a href="http://build.ros.org/{{ build.uri }}" target="_blank">
+                                <span class="glyphicon glyphicon-{{ build.icon }}"></span>
+                                <span style="font-weight: bold; padding-left: 10px;">#{{ build.id }}</span>
```

Comment:
Please move this styling into the css for consistency. And can we enable color changing based on the success status?

Reply:
Done. Please see comment below commit.